Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -364,7 +364,7 @@ public class PBXProjGenerator {
 
         for dependency in target.dependencies {
 
-            let embed = dependency.embed ?? (target.type.isApp ? true : false)
+            let embed = dependency.embed ?? target.shouldEmbedDependencies
```

Comment:
Oh maybe I embedded the bug here. Thanks👍
I was about to add Quick.framework via Carthage to my app's test target.